config/jobs/kubernetes/wg-k8s-infra/trusted/wg-k8s-infra-trusted.yaml

```
@@ -87,13 +87,23 @@ periodics:
       args:
       - -c
       - |
-        set -euo pipefail; \
```

removed since `snyk test` might return 1 if vulns found

---

however, this may silence any errors occurred earlier during `apt update` or while installing jq or snyk

---

updated the PR
- keep `set -euo pipefail` and update the `snyk test` command with exit code check